box.json

```diff
@@ -1,23 +1,11 @@
 {
     "alias": "doctrine-migrations.phar",
-    "compactors": [
```

Comment:
I don't think it's gonna make a big difference, but you can keep the compactors, I would replace `Composer` with `Php` though